elastic-buffer-sim/src/Bittide/ClockControl.hs

```diff
@@ -75,15 +75,15 @@ instance KnownNat n => ToField (Unsigned n) where
 instance KnownNat n => ToJSON (Unsigned n) where
   toJSON = toJSON . toInteger
 
-defClockConfig :: ClockControlConfig
+defClockConfig :: ClockControlConfig 16
 defClockConfig = ClockControlConfig
   { cccPessimisticPeriod = pessimisticPeriod
   -- clock adjustment takes place at 1MHz, clock is 200MHz so we
   -- can have at most one correction per 200 cycles
   , cccSettlePeriod      = pessimisticPeriod * 200
   , cccDynamicRange      = 150
   , cccStepSize          = 1
-  , cccBufferSize        = 65536 -- 128
+  , cccBufferSize        = d16 -- 2**16 ~ 65536
```

Comment:
Good question, I presume they're picked to make sure all the example topologies we have don't run out of bounds. Once we add boot mode this will be reduced to something much more sensible (say, `d5` for 32 elements).